stackabletech/test-dev-cluster
# Feature/add kafka operator support

## The problem

The Kafka-Operator requires Zookeeper.

## Proposed solution

Install Zookeeper in sidecar containers. Maybe use the Zookeeper operator for the installation.

## How to test

Since there is no `kafka-operator-integration-tests` repository at this moment you need to create an empty folder for now:
```
mkdir kafka-operator-integration-tests
```

Start a test cluster:
```
./init.sh debian kafka-operator
```
Check that a single node Zookpeer cluster has been installed: 
```
docker exec -it debian_sidecar_1 kubectl get zookeeperclusters
```
Run the Kafka operator
```
./run.sh run-operator
```


closes #21
